scipy/optimize/_qap.py

```diff
@@ -308,48 +317,36 @@ def _quadratic_assignment_faq(A, B,
         instance, then that object is used.
         Default is None.
 
-    partial_match : 2d-array of integers, optional, (default = None)
-        Allows the user to fix part of the matching between the two
-        matrices. In the literature, a partial match is also known as a
-        "seed".
-
-        Each row of `partial_match` specifies the indices of a pair of
-        corresponding nodes, that is, node ``partial_match[i, 0]`` of `A` is
-        matched to node ``partial_match[i, 1]`` of `B`. Accordingly,
-        ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
-        not greater than the number of nodes, :math:`n`.
-
-    P0 : 2d-array, "barycenter", or "randomized" (default = "barycenter")
-        The initial (guess) permutation matrix or search "position"
-        `P0`.
+    P0 : 2-D array, "barycenter", or "randomized" (default: "barycenter")
+        The initial (guess) permutation matrix or search "position".
 
         `P0` need not be a proper permutation matrix;
         however, it must be :math:`m' x m'`, where :math:`m' = n - m`,
         and it must be doubly stochastic: each of its rows and columns must
         sum to 1.
 
         If unspecified or ``"barycenter"``, the non-informative "flat
-        doubly stochastic matrix" :math:`J = 1*1^T/m'`, where :math:`1` is
-        a :math:`m' \times 1` array of ones, is used. This is the "barycenter"
-        of the search space of doubly-stochastic matrices.
+        doubly stochastic matrix" :math:`J = \mathbf{1} \cdot \mathbf{1}^T/m'`,
+        where :math:`1` is a :math:`m' \times 1` array of ones, is used. This
```

Comment:
```suggestion
        where :math:`\mathbf{1}` is a :math:`m' \times 1` array of ones, is used. This
```
Not sure if this will put you over the line limit.

Reply:
I went with a different approach: `This is a :math: m' x m' matrix with all entries equal to :math:1 / m'`.`

This is from [a definition](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.214.3184&rep=rep1&type=pdf):
`The barycenter of B_n is the van der Waerden constant matrix B_∗ with all entries equal to 1/n`

Reply:
Sounds good.